src/pages/PdfAuditResultsPage.tsx

```
@@ -236,6 +263,25 @@ export const PdfAuditResultsPage: React.FC = () => {
+    const SEVERITY_WEIGHT: Record<string, number> = { critical: 4, serious: 3, moderate: 2, minor: 1 };
+    const sorted = auditResult.issues
+      .map((issue, originalIndex) => ({ issue, originalIndex }))
+      .sort((a, b) => {
+        const pageDiff = (a.issue.pageNumber ?? 0) - (b.issue.pageNumber ?? 0);
+        if (pageDiff !== 0) return pageDiff;
+        const weightDiff = (SEVERITY_WEIGHT[b.issue.severity] ?? 0) - (SEVERITY_WEIGHT[a.issue.severity] ?? 0);
+        if (weightDiff !== 0) return weightDiff;
+        return a.originalIndex - b.originalIndex;
+      });
+    const map = new Map<string, number>();
+    sorted.forEach(({ issue }, idx) => map.set(issue.id, idx + 1));
```

Biome (2.4.6)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)